modules/bmc_check/esbmccheck.py

```diff
@@ -793,28 +791,26 @@ def kinductioncheck(self, _cprogrampath):
         if self.debug:
             print(">> Starting the verification of the P\' program")
 
-        #try:
-        #    file = open(_cprogrampath, "r")
-        #    lines = file.readlines()
-        #    res = [y for y in lines if "safe_write" in y or "fflush_all" in y or "bb_show_usage" in y or "bb_error_msg_and_die" in y]
```

Comment:
Do we now support the busybox benchmarks?

Reply:
Yes.